### DISCUSSION TITLE
what the happened?

### DISCUSSION THREAD

**Discussion by smccjjj** [veighna-global/vnpy_evo#15] [Q&A]
我链接到币安交易所会出现这种情况。
![微信截图_20220920145815](https://user-images.githubusercontent.com/83157082/191190561-54384c37-ea79-47d0-ab22-97efe74afb98.png)

`
这个是什么情况？ 币安实盘也没链接上。行情等都没链接上

**Comment by smccjjj** [comment]
![微信截图_20220920153532](https://user-images.githubusercontent.com/83157082/191195962-fec47058-0ced-4dae-ba18-959ef8887fdf.png)

**Comment by smccjjj** [comment]
有人吗  试了下okx 其他交易所都是报错这个 有本地window +vpn跑起来的小伙伴吗 指导指导

**Comment by xldistance** [comment]
你要实在不会用代理就买个境外云跑
